bin/CoreData/Shaders/GLSL/v2/_Shadow.glsl

```diff
@@ -85,17 +85,20 @@ vec3 DirectionToUV(const vec3 vec, const vec2 bias)
     /// Sample shadow map texture at given 4-coordinate
     half SampleShadow(const vec4 shadowPos)
     {
-        #if defined(GL3)
-            return textureProj(sShadowMap, shadowPos);
-        #elif defined(GL_ES)
+        #if defined(GL_ES)
             return texture2DProj(sShadowMap, shadowPos).r * shadowPos.w > shadowPos.z ? 1.0 : 0.0;
+        #elif defined(GL3)
+            return textureProj(sShadowMap, shadowPos);
         #else
             return shadow2DProj(sShadowMap, shadowPos).r;
         #endif
     }
 
     /// Sample shadow map texture with given offset
-    #if defined(GL3)
+    #if defined(GL_ES)
+        #define SampleShadowOffset(shadowPos, identity, dx, dy) \
+            SampleShadow((shadowPos) + vec4(identity.x * float(dx), identity.y * float(dy), 0.0, 0.0))
+    #elif defined(GL3)
```

Comment:
ow. Let's not copypaste things
`#if defined(GL3) && !defined(GL_ES)` for the first branch will do fine